src/python/espressomd/particle_data.pyx

```diff
@@ -1022,11 +961,7 @@ cdef class ParticleHandle(object):
                     """
                     The particle translational frictional coefficient used in the Langevin thermostat.
 
-                    gamma_rot : list of floats
-
-                    .. note::
-
-                       This needs the feature LANGEVIN_PER_PARTICLE, ROTATION and ROTATIONAL_INERTIA
+                    gamma_rot : list of floats (if using feature ROTATIONAL_INTERIA)
```

Comment:
The feature ROTATIONAL_INERTIA is required anyways

Reply:
I was trying to differentiate between the float gamma_rot and the vector, which depends on the ROTATIONAL_INERTIA feature. I guess  the documentation build will take care of that?